opensrp-reporting/src/main/java/org/smartregister/reporting/impl/ReportingUtil.java

```diff
@@ -0,0 +1,76 @@
+package org.smartregister.reporting.impl;
+
+import android.content.Context;
+import android.graphics.Color;
+import android.support.annotation.Nullable;
+import android.view.View;
+
+import org.smartregister.reporting.R;
+import org.smartregister.reporting.domain.IndicatorTally;
+import org.smartregister.reporting.domain.PieChartSlice;
+import org.smartregister.reporting.domain.ReportingIndicatorVisualization;
+import org.smartregister.reporting.impl.models.IndicatorModel;
+import org.smartregister.reporting.impl.models.PieChartViewModel;
+import org.smartregister.reporting.impl.views.IndicatorView;
+import org.smartregister.reporting.view.IndicatorVisualisationFactory;
+
+import java.util.List;
+import java.util.Map;
+
+import static org.smartregister.reporting.util.AggregationUtil.getLatestIndicatorCount;
+import static org.smartregister.reporting.util.AggregationUtil.getStaticIndicatorCount;
+
+public class ReportingUtil {
+
+
+    // Color definitions for the chart slices. This could essentially be defined in colors.xml
+    public static final int YES_GREEN_SLICE_COLOR = Color.parseColor("#99CC00");
+    public static final int NO_RED_SLICE_COLOR = Color.parseColor("#FF4444");
+
+    public static long getTotalStaticCount(List<Map<String, IndicatorTally>> indicatorTallies, String indicatorKey) {
```

Comment:
What does static count mean?

Reply:
Static to mean fixed count. Not sure, if that's the right choice of word but it is not the most recent count based on date and time. Whats your suggestion??